interface/main/calendar/add_edit_event.php

```diff
@@ -81,7 +87,7 @@
 if (isset($_GET['starttimeh'])) {
```

Comment:
In this script, it looks like $starttimeh feeds the form_hour form element. Are we sure this doesn't break anything when form is submitted and doesn't break any of the related javascript stuff in this script?

Reply:
I made some changes to this for the am/pm fix.  From what I can see it doesnt break it it, however I'm not the best with javascript.